PYTHON_CDM_Conversion_code/utils.py

```diff
@@ -24,12 +26,23 @@
 SUBDAILY_CDM_LITE_OUT_DIR = config.get("Paths", "subdaily_cdmlite_outdir")
 SUBDAILY_CDM_OBS_OUT_DIR = config.get("Paths", "subdaily_cdmobs_outdir")
 SUBDAILY_CDM_HEAD_OUT_DIR = config.get("Paths", "subdaily_cdmhead_outdir")
+DAILY_CSV_IN_DIR = config.get("Paths", "daily_csv_indir")
+DAILY_HEAD_IN_DIR= config.get("Paths", "daily_head_indir")
+DAILY_CDMLITE_OUT_DIR = config.get("Paths", "daily_cdmlite_outdir")
+DAILY_OBS_OUT_DIR = config.get("Paths", "daily_obs_outdir")
+DAILY_QC_OUT_DIR = config.get("Paths", "daily_qc_outdir")
+DAILY_CDMHEAD_OUT_DIR= config.get("Paths", "daily_head_outdir")
 # Files
 SUBDAILY_CDM_LITE_FILE_ROOT = config.get("Filenames", "subdaily_cdmlite_file")
 SUBDAILY_QC_FILE_ROOT = config.get("Filenames", "subdaily_qc_file")
 SUBDAILY_CDM_OBS_FILE_ROOT = config.get("Filenames", "subdaily_cdmobs_file")
 SUBDAILY_CDM_HEAD_FILE_ROOT = config.get("Filenames", "subdaily_cdmhead_file")
+DAILY_OBS_FILE_ROOT = config.get("Filenames", "daily_cdmobs_file")
+DAILY_CDMLITE_FILE_ROOT = config.get("Filenames", "daily_cdmlite_file")
+DAILY_CDMHEAD_FILE_ROOT= config.get("Filenames", "daily_cdmhead_file")
 # Station records (note there are two different record_id .csv files needed one for
 # observations tables and one for the header table
 STATION_RECORD_ENTRIES_OBS_LITE = config.get("Records", "station_records_obs_lite")
```

Comment:
Should probably correct these to "SUBDAILY_STATION_REC...." for consistency (and of course in the relevant files too.  If easier, could make a new issue to do this minor change just to keep things clear.

Reply:
I fixed all of those oustanding issues and pushed the changes . sorry i missed them , covid brain !!:(